zerver/tests/test_markdown.py

```diff
@@ -538,13 +538,13 @@ def test_inline_youtube(self) -> None:
         )
 
         msg = (
-            "https://www.youtube.com/playlist?v=O5nskjZ_GoI&list=PL8dPuuaLjXtNlUrzyH5r6jN9ulIgZBpdo"
+            "https://www.youtube.com/watch?v=O5nskjZ_GoI&list=PL8dPuuaLjXtNlUrzyH5r6jN9ulIgZBpdo"
```

Comment:
I think this URL is intended to be a playlist; we should just pick a different one if we want a /watch URL.

Reply:
Whatever the intention was, the old URL does not work—if you visit it, it opens the playlist without retaining any reference to the specific video.

I suspect it was generated by hand as a test case for the old code that treated `/playlist` and `/watch` as similar, without checking whether the test actually made sense.